### ISSUE TITLE
Mod Review

### ISSUE THREAD

**Issue by Bysokar** [StargateMC/StargateMCPublic#274]
**The following mods need to be reviewed, to verify they really do benefit the game in some way, or be removed:**

- Advanced Solar Panels (Primarily looking to remove this due to server-crashing incompatibilities with WarpDrive)
- Minecraft Comes Alive!
- Dons Lightning Rod Mod

**The following mods should be investigated and tested before possibly being added to the game:**
- ArchitectureCraft
- Computronics

**Comment by Maxstripe** [comment]
MCA doesnt add much of value. NPC slaves can mine and cook but arent useful for much else except making lag. that can go without affecting the server too much
dons lighting rod doesnt work at all (followed instructions to make it work, made lighting hit it, didnt produce power) can go with no issues as no one uses it

**Comment by PixelGamerX32** [comment]
I have tried the 'cooking' feature from the M.C.A. mod myself offline not on the server, but it does not really work that well. 
But it does make the villages more realistic than just Minecraft vanilla generic villagers.
But what will happen to the recipes if the Advanced Solar Panels mod is removed?
It could cause some recipes to become unusable and may need to be changed to compensate.

**Comment by XredstonemodsX** [comment]
MCA and Don's can go, however Advanced Solar Panels is pretty great and shouldn't be removed.

Also can you add Big Reactors to the 'Pending review' list?

**Comment by Bysokar** [comment]
No, I'm not looking to add any new mods until after the content phase things is done.
As for Advanced Solar Panels - if I am not able to resolve the crash bug when warp drive ships jump with them I will be looking to remove it.
